src/index.ts

```diff
@@ -1,27 +1,96 @@
-import { CommandoClient } from 'discord.js-commando';
-import path from 'path';
-import dotenv from 'dotenv';
-import { Activity } from 'discord.js';
-
-dotenv.config();
-
-const client = new CommandoClient({
-  commandPrefix: '.nick',
-  presence: {
-    status: "dnd"
-  
-  }
+const commando = require('discord.js-commando');
```

Comment:
Please just do
`import * as commando from 'discord.js-commando'`

Reply:
I left this as require on purpose, as the library had problems with import here too

Reply:
I think its still possible, give it a try and tell me how it goes

Reply:
https://github.com/NewCircuit/nickreq/commit/ee374c4c3fffa75fd7df9357c5d7a74ff4246ef9#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R4

Reply:
yep, same issue, import doesnt work with it